packages/dd-trace/src/llmobs/telemetry.js

```diff
@@ -1,12 +1,54 @@
 'use strict'
 
+const {
+  SPAN_KIND,
+  MODEL_PROVIDER,
+  PARENT_ID_KEY,
+  SESSION_ID,
+  ROOT_PARENT_ID,
+  INTEGRATION,
+  DECORATOR
+} = require('./constants/tags')
+
 const telemetryMetrics = require('../telemetry/metrics')
+
+const LLMObsTagger = require('./tagger')
+
 const llmobsMetrics = telemetryMetrics.manager.namespace('mlobs')
 
 function incrementLLMObsSpanStartCount (tags, value = 1) {
   llmobsMetrics.count('span.start', tags).inc(value)
 }
 
+function incrementLLMObsSpanFinishedCount (span, value = 1) {
+  const mlObsTags = LLMObsTagger.tagMap.get(span)
+
+  const isRootSpan = mlObsTags[PARENT_ID_KEY] === ROOT_PARENT_ID
```

Comment:
saw you changed this in the last commit from `!==` to `===`

just confirming we'll need to change on the python side too?
```python
is_root_span = span._get_ctx_item(PARENT_ID_KEY) != ROOT_PARENT_ID
```

Reply:
ugh i didn't realize this mistake was also in the python side. Yeah the root span check needs to verify that the parent ID is set to `undefined`, I'll make the corresponding fix on the python side too